definitions/divorce/json/AuthorisationCaseState.json

```diff
@@ -1273,34 +1273,13 @@
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "R"
   },
-  {
-    "LiveFrom": "25/06/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseStateID": "DNPronounced",
-    "UserRole": "caseworker-divorce-courtadmin-la",
-    "CRUD": "RU"
-  },
-  {
-    "LiveFrom": "25/06/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseStateID": "DNPronounced",
-    "UserRole": "caseworker-divorce-courtadmin_beta",
-    "CRUD": "RU"
-  },
   {
     "LiveFrom": "25/06/2019",
     "CaseTypeID": "DIVORCE",
     "CaseStateID": "DNPronounced",
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "R"
   },
-  {
-    "LiveFrom": "25/06/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseStateID": "DNPronounced",
-    "UserRole": "caseworker-divorce-superuser",
-    "CRUD": "RU"
-  },
```

Comment:
Not 100% sure which permissions should stay, check the existing entry (some have only R)